frontend/src/app/containers/ExperimentPage/DetailsOverview.js

```diff
@@ -84,24 +84,34 @@ export default function DetailsOverview({
 export const LaunchStatus = ({ experiment, graduated }: LaunchStatusType) => {
   const { created, completed } = experiment;
 
-  const completedDate = formatDate(completed);
+  const completedDate = new FluentDateTime(completed, {
+    day: "numeric",
+    month: "long",
+    year: "numeric"
+  });
+
   if (graduated) {
     return (
-      <LocalizedHtml id="completedDateLabel" $completedDate={completedDate}>
+      <Localized id="completedDateLabel" $completedDate={completedDate} b={<b></b>}>
         <span>
-          Experiment End Date: <b>{completedDate}</b>
+          Experiment End Date: <b>{completed}</b>
         </span>
-      </LocalizedHtml>
+      </Localized>
     );
   }
 
-  const startedDate = formatDate(created);
+  const startedDate = new FluentDateTime(created, {
```

Comment:
@flodolo Now that `startedDate` is a `FluentDateTime` object, do you think we should use the `DATETIME` builtin explicitly in the FTL? Like so:

```diff
-startedDateLabel = Experiment start date: <b>{$startedDate}</b>
+startedDateLabel = Experiment start date: <b>{DATETIME($startedDate)}</b>
```

I don't even think we need to rev the id for this, but I'll leave the decision up to you.

Reply:
If we do, we either need to rev the ID, or change the existing translation via script.

Reply:
The current value of 

```properties
startedDateLabel = Experiment start date: <b>{$startedDate}</b>
```

will continue to work, because Fluent recognizes the argument as a `FluentDateTime` anyways.

Reply:
That's good, but nobody will realize they can actually customize the format if we don't add `DATETIME()`.

Reply:
That's a good point. Regardless of whether we need to rev the id, or use a script, would you rather use the version with `DATETIME()` or without?

Reply:
I think it's good to have DATETIME(), it would be easy to help a locale customize that if needed.